forest/poplar/legacy/common_funcs.py

```diff
@@ -173,10 +173,10 @@ def read_data(beiwe_id: str, study_folder: str, datastream: str, tz_str: str,
         all_timestamps: list = []
 
         for i in directories:
-            filenames, filestamps = get_files_timestamps(
+            directory_filenames, directory_filestamps = get_files_timestamps(
                 os.path.join(study_folder, beiwe_id, i)
             )
-            all_timestamps = all_timestamps + filestamps.tolist()
+            all_timestamps = all_timestamps + directory_filestamps.tolist()
```

Comment:
Simpler: `all_timestamps += directory_filestamps.tolist()`

Reply:
I generally avoid using += on lists in python (https://stackoverflow.com/questions/2347265/why-does-behave-unexpectedly-on-lists), but I suppose that in this instance, += shouldn't cause any unexpected behavior, correct?

Reply:
Well, that's what tests are for? :) Also, in this case you're not using a class.